instagram-app/src/components/PostContainer/PostsPage.jsx

```diff
@@ -0,0 +1,21 @@
+import React from "react";
+import { Row, Col } from "reactstrap";
+import PostContainer from "./PostContainer";
+
+const PostsPage = ({ data }) => {
+  return (
+    <Row>
+      <Col sm="12" md={{ size: 6, offset: 3 }}>
+        {data.map((post, index) => {
+          return (
+            <div className="post-container" key={index}>
```

Comment:
This is fine but you could as well use a library as uuid to create a unique id. Using indexes is ok in this kind of small projects but is usually not recommended.